xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -41,6 +41,9 @@
      <p>{$pay_later_receipt}</p> {* FIXME: this might be text rather than HTML *}
     {/if}
 
+    <p>{ts}With kind regards,{/ts}    
+    {domain.name}</p>
```

Comment:
I don't know about "With kind regards" as it's fairly culturally specific.  @bgm @bjendres @sluc23 Your thoughts please?

Reply:
Valid point, but I believe there are many equivalents in the different languages for this. See also: https://www.indifferentlanguages.com/words/with_kind_regards

Reply:
I left this out of this PR and will submit the courtesy changes in another pr.